.gitpod.yml

```diff
@@ -0,0 +1,21 @@
+---
+image:
+  file: container/gitpod/Dockerfile
+
+tasks:
+  - name: Lint the Javascript dependencies
+    before: npm i
+    command: npm run lint
+  - name: Run the fullstack tests
+    before: |
+      bash -x tools/ci/build_cache.sh
+      bash -x tools/ci/build_autoinst.sh
+      sudo chown -R gitpod $(pwd)
+    command: tools/ci/run_unit_tests.sh fullstack
```

Comment:
this selection of tasks looks rather arbitrary. By default everything should be accessible over the central Makefile. So e.g. `make test` should do something useful after initial dependencies have been installed. I would refrain from calling any scripts from tools/ci as that directory is meant exactly for only CI, the continuous integration systems, e.g. circleCI, or GHA, but not an interactive development environment. However we can make the provisioning of os-autoinst more easily accessible. And maybe it's as easy as renaming tools/ci/build_autoinst.sh to tools/build_autoinst.sh :) But why for example do you need tools/ci/build_cache.sh?

Reply:
Yes, the selection of tasks is very arbitrary: I have simply copy and pasted it from the `ci.yml` worklfow.

Which tasks should be run here?

Reply:
I already explained that above: `make test`

Reply:
> I have simply copy and pasted it from the ci.yml worklfow

@dcermak This is only a subset because our main tests are on CircleCI. And it assumes that you can choose the target via GitHub UI e.g. `checkstyle`, `t`, `ui`, `api`, `fullstack`, `fullstack-unstable`.

If one can select it here as well, that would be really cool - but I'd say all tests is also fine for now.